cli/cmd/report/report.go

```diff
@@ -2,57 +2,94 @@ package report
 
 import (
 	"errors"
-	"fmt"
+	"os"
 	"os/exec"
 	"runtime"
 
+	"numerous/cli/cmd/output"
+
 	"github.com/spf13/cobra"
 )
 
 const numerousReportURL = "https://numerous.com"
 
+var (
+	ErrWSLCheck        = errors.New("error when checking windows subsystem for linux")
+	ErrOSCheck         = errors.New("error to identify the current OS")
```

Comment:
Just a small grammar thing here. Instead it could be just be a short `"error identifying OS"`